src/cljdoc/render/search.clj

```diff
@@ -0,0 +1,32 @@
+(ns cljdoc.render.search
+  (:require [cljdoc.render.layout :as layout]
+            [cljdoc.util :as util]))
+
+(defn search-form
+  ([] (search-form ""))
+  ([search-terms]
+   [:div.w-90.mb4
+    [:div#cljdoc-search {:initial-value search-terms}]]))
```

Comment:
Maybe `data-initial-value` is more appropriate here, but again, not very important.

Reply:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes